Framework/PythonInterface/plugins/functions/ChudleyElliot.py

```diff
@@ -27,10 +27,14 @@
 import math
 import numpy as np
 from mantid.api import IFunction1D, FunctionFactory
+from scipy import constants
 
 
 class ChudleyElliot(IFunction1D):
 
+    planck_constant = constants.Planck / constants.e * 1E15  # meV*psec
```

Comment:
`constants.value('Planck constant over 2 pi in eV s') * 1e15` may be a cleaner way to express this.

Reply:
That is cleaner. I didn't know about the natural language feature of scipy constants, nice.